draft-ietf-rats-msg-wrap.md

```diff
@@ -296,10 +296,9 @@ CMW itself provides no facilities for authenticity, integrity protection, or con
 It is the responsibility of the designer for each use case to determine the necessary security properties and implement them accordingly.
 A secure channel (e.g., via TLS) or object-level security (e.g., using JWT) may suffice in some scenarios, but not in all.
 
-When a CMW is used to carry the Evidence for composite or layered attestation for a single device, the security properties needed are that of attestation.
-In particular, all the members in a CMW must be bound together so that an attacker can not replace one Evidence message showing compromise with that from a non-compromised device.
-The authenticity and integrity protection MUST be attestation-oriented.
-For further security considerations about collections, see {{seccons-coll}}.
+When a CMW is used to carry Evidence for composite or layered attestation of a single device, all components within the CMW must be cryptographically
+bound to prevent an attacker from replacing compromised Evidence with that of a non-compromised device. The protection of authenticity and integrity
```

Comment:
The problem is not with "compromised evidence", but rather with "evidence from a compromised device"

```suggestion
bound to prevent an attacker from replacing Evidence from a compromised device with Evidence from a non-compromised device. The protection of authenticity and integrity
```